addons/l10n_ph/utils.py

```diff
@@ -50,24 +52,29 @@ def _export_bir_2307(sheet_title, moves, file_format='xlsx'):
     for move in moves:
         partner = move.partner_id
```

Comment:
Maybe we should just take all the partner information from the `commercial_partner_id`?
```suggestion
        partner = move.commercial_partner_id
```

Reply:
Hello @kidoodoo 👋🏻
What’s your opinion on this — should we go ahead with this change ??

Reply:
Good afternoon, @prep-odoo, 
Yes, in terms of the requirement, fetching below info based on commercial_partner_id works as long as you maintain the same conditions we discussed.

- "Vendor_TIN": "vat",
- "branchCode": "branch_code",
- "companyName": "company_name",
- "surName": "last_name",
- "firstName": "first_name",
- "middleName": "middle_name",
- "address": "address",
- "zip_code": "zip"